org.eclipse.transformer.cli/src/main/java/org/eclipse/transformer/jakarta/JakartaTransformer.java

```diff
@@ -36,6 +36,7 @@ public static void main(String[] args) throws Exception {
 	public static final String	DEFAULT_BUNDLES_REFERENCE		= "jakarta-bundles.properties";
 	public static final String	DEFAULT_DIRECT_REFERENCE		= "jakarta-direct.properties";
 	public static final String	DEFAULT_MASTER_TEXT_REFERENCE	= "jakarta-text-master.properties";
+        public static final String	DEFAULT_PER_CLASS_CONSTANT_MASTER_REFERENCE	= "jakarta-per-class-constant-master.properties";
```

Comment:
The formatting is wrong. The formatter uses tabs and not space. Not to get into a war, but we should be consistent so that future changes don't reformat the code.

This comment applies to all the files in this PR.

Reply:
Thank-you, I reformatted all the changes.